docs/installation/certificate.asciidoc

```diff
@@ -203,147 +172,136 @@ systemctl restart packetfence-haproxy-admin
 systemctl restart packetfence-haproxy-portal
 ----
 
-=== 4. Implementation of SSL/TLS Certificate for RADIUS
+=== Implementation of SSL/TLS Certificate for RADIUS
 
-==== 4.1 Generate CSR
+==== Generate CSR
 
-If you already have a certificate for your captive portal, you can use the same certificate for RADIUS. In this case, please go directly to section 4.3 Install the SSL/TLS Certificate on the Server.
+If you already have a certificate for your captive portal, you can use the same certificate for RADIUS. In this case, please go directly to section
+link:https://www.packetfence.org/docs/PacketFence_Installation_Guide.html#_install_the_ssltls_radius_certificate_on_the_server[36.4.3 Install the SSL/TLS].
 
 Warning: Wildcard certificates will not work with RADIUS. If you are using a wildcard certificate for your captive portal, you will need a new certificate specifically for RADIUS.
 
-To generate a Certificate Signing Request (CSR) for RADIUS, use the following command:
+To generate a CSR, follow these steps:
+
+- Log on the admin web interface (GUI)
+
+- Go to Configuration > System Configuration > SSL Certificates > RADIUS
+
+image::certificate/15-Radius-CSR.png[scaledwidth="100%",alt="Radius-CSR"]
 
+- Click on “Generate Signing Request (CSR)
+
+- Complete the following using your own information, Common Name must match the captive portal hostname in Configuration > General Configuration > Hostname
+
+image::certificate/17-CSR-radius.png[scaledwidth="100%",alt="CSR"]
+
+- Save the CSR and private key to a secure location, you will need it to renew your certificate.
+
+- You can also generate the CSR with CLI using:
 [source, shell]
 ----
-( _fd="/usr/local/pf/raddb/certs/server.key" ; _fd_csr="/usr/local/pf/raddb/certs/MYCSR.csr" ; openssl req -new -sha256 -key ${_fd} -out ${_fd_csr} -config <(
-cat <<-EOF
-[req]
-default_bits = 2048
-prompt = no
-default_md = sha256
-req_extensions = req_ext
-distinguished_name = dn
-
-[ dn ]
-C=Country
-ST=State
-L=Locality
-O=Organization
-CN=Common Name
-
-[ req_ext ]
-subjectAltName = @alt_names
-extendedKeyUsage = serverAuth
-
-[ alt_names ]
-DNS.1 = pf.domain.lan
-DNS.2 = packetfence.domain.lan
-EOF
-))
+ openssl req -newkey rsa:2048 -keyout PRIVATEKEY.key -out MYCSR.csr
 ----
 
-==== 4.2 Submit the CSR to a Certificate Authority (CA)
+==== Submit the CSR to a Certificate Authority (CA)
 
-Please refer to section 2.2 Submit the CSR to a Certificate Authority (CA) and follow the steps mentioned there.
+Please refer to section 
+link:https://www.packetfence.org/docs/PacketFence_Installation_Guide.html#_submit_the_csr_to_a_certificate_authority_ca[36.2.2 Submit the CSR to a Certificate Authority (CA)]. and follow the steps mentioned there.
```

Comment:
I think this link will not work. I was not able to find the anchor (related to title or subtitle, if I remember correctly)